lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runtime/azbatch/client/AzureBatchDriverConfigurationProviderImpl.java

```diff
@@ -76,6 +76,11 @@
     this.commandBuilder = commandBuilder;
   }
 
+  /**
+   * Assembles the Driver configuration.
+   *
```

Comment:
Missing parameters in java doc.

I thought Reef will have some maven plugin to check Java doc. But seems not.